k8s/parser.go

```diff
@@ -0,0 +1,150 @@
+package k8s
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/goark/go-cvss/v3/metric"
+
+	version "github.com/aquasecurity/go-pep440-version"
```

Comment:
Isn't [this](https://github.com/aquasecurity/go-version/tree/main/pkg/versi) the package we need rather than PEP440?